.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,26 @@
+name: govulncheck
+
+on:
+  push:
```

Comment:
I'm not sure we need this on push.  I think it should be fine on PR, as we'll catch everything as it comes in.